lib/connection.js

```diff
@@ -1582,7 +1582,7 @@ Connection.prototype.syncIndexes = async function syncIndexes(options = {}) {
 };
 
 /**
- * Switches to a different database using the same connection pool.
+ * Switches to a different database using the same [connection pool](../connections.html#connection_pools).
```

Comment:
```suggestion
 * Switches to a different database using the same [connection pool](https://mongoosejs.com/docs/api/connectionshtml#connection_pools).
```

since #13328 in code we use absolute paths so that the URLs are valid both in-editor and in the website